Software/app/freefall_lorawan/sys_app.c

```diff
@@ -73,12 +73,8 @@ void SystemApp_Init(void)
   /* Configure the debug mode*/
   DBG_Init();
 
-  /*Initialize the terminal */

```

Comment:
Maybe keep this line (with added space)?

Reply:
Done